src/main/java/frc/team2412/robot/commands/intake/FeederInCommand.java

```diff
@@ -15,14 +15,4 @@ public FeederInCommand(IntakeSubsystem intakeSubsystem) {
 	public void initialize() {
 		intakeSubsystem.feederIn();
 	}
-
-	@Override
-	public void end(boolean interrupted) {
-		intakeSubsystem.feederStop();
-	}
-
-	@Override
-	public boolean isFinished() {
-		return intakeSubsystem.feederSensor();
-	}
```

Comment:
This part should be kept (same for `IndexInCommand` and `IntakeInCommand`) - what James and I meant was deleting `isFinished()` from the commands where it just returns `false`, like `FeederOutCommand`. Sorry for any confusion earlier.

Reply:
Looking at [Command::isFinished](https://github.com/wpilibsuite/allwpilib/blob/da3abade83ada38d387d7ab0633f0914a44ff9d1/wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Command.java#L62), it returns `false` by default. I thought it returned `true`. So, the `isFinished()` override needs to be kept, and just updated to return `true` so the diagnostic command works correctly. Sorry about the confusion.

Reply:
I'm fine with making commands like `IntakeStopCommand` and `IntakeOutCommand` return `true`, but for the "in" commands (`IndexInCommand`, `IntakeInCommand`, and `FeederInCommand`) I think we should keep `isFinished()` as is (checking the sensors) since the intent was to make the command run until a note is detected, which is additional logic I don't think we should compromise for a diagnostic command.

To keep the diagnostic command from being stuck waiting for a note, I'd suggest adding [`withTimeout()`](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj2/command/Command.html#withTimeout(double)) to make the in commands only run for a given period of time. (I could also see some value in keeping in commands that use the sensors, just to verify that the sensor hardware is working correctly).

Reply:
Seems reasonable.